### PR TITLE
[bitnami/kafka] Add provisioning job tolerations

### PR DESCRIPTION
**Pull request by samgh96**
Signed-off-by: Samuel García <samgh96@gmail.com>

### Description of the change

This PR adds tolerations to the job created whenever `provisioning` is set to `true`.

### Benefits

Being able to configure tolerations to the provisioning job.

### Possible drawbacks

N/A

### Applicable issues

N/A

### Additional information

N/A

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/kafka/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 16.3.0
+version: 16.3.1
```
Comment (marcosbc) [inline bitnami/kafka/Chart.yaml:32]: Since this is adding a new feature, could we increase the minor instead? E.g.:

```suggestion
version: 16.4.0
```

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please have a look at my comments.

File: bitnami/kafka/Chart.yaml (hunk 1 above)
Comment (samgh96) [reply bitnami/kafka/Chart.yaml]: Done!

**Review by marcosbc — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**